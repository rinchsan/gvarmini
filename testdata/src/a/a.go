package a

var _, _, _, _ int // want "多いよ"

func f() {}
