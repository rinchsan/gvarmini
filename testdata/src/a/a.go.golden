package a

var (
    _ int
    _ int
    _ int
    _ int
) // want "多いよ"

func f() {}
